src/ipsec/_ipsec.in

```diff
@@ -257,7 +257,17 @@ stop)
 			loop=110
 			while [ $loop -gt 0 ] ; do
 				kill -0 $spid 2>/dev/null || break
-				sleep 0.1 2>/dev/null
+				if [ "$(readlink $(which sleep))" = "busybox" ]
+				then
+					if [ -n "$(which usleep") ]
+					then
+						usleep 100000 2>/dev/null
+					else
+						sleep 1 2>/dev/null
+					fi
+				else
+					sleep 0.1 2>/dev/null
+				fi
```

Comment:
I think it's OK as was before.

The whole point is the `if` below is as follows (note redirection of stderr to `/dev/null` to hide error messages):
- if sleep returns error, probably because of no support for fractional values, wait one second and decrease `$loop` by 10,
-  otherwise we waited 0.1 s so decrease `$loop` by 1.
That way we have 100-millisecond granularity if `sleep` supports fractional numbers, if does not - wait one second before next check.

Reply:
Okay, dropping this change.